.clang-format

```diff
@@ -80,10 +80,11 @@ IncludeCategories:
 IncludeIsMainRegex: '(Test)?$'
 IncludeIsMainSourceRegex: ''
 IndentCaseLabels: false
-IndentGotoLabels: true
+IndentGotoLabels: false
 IndentPPDirectives: None
 IndentWidth:     2
 IndentWrappedFunctionNames: false
+IndentExternBlock: false
```

Comment:
Many headers are wrapped with `extern "C"` blocks, don't indent the code found inside those.